README.md

```diff
@@ -65,27 +65,27 @@ Launch the workspace by running the following from the command line:
 
 This will ensure any dependencies are ready before launching Xcode. 
 
-You can also open the project by double clicking on WordPress.xcworkspace file, or launching Xcode and choose `File` > `Open` and browse to `WordPress.xcworkspace`.
+You can also open the project by double clicking on `WordPress.xcworkspace` file, or launching Xcode and choose `File` > `Open` and browse to `WordPress.xcworkspace`.
 
 ### Setup Credentials
 
-In order to login to WordPress.com using the app you will need to create an account over at the [WordPress.com Developer Portal](https://developer.wordpress.com).
+In order to login to WordPress.com with the app you need to create an account over at the [WordPress.com Developer Portal](https://developer.wordpress.com).
 
-After you created an account you can create an application on the [WordPress.com applications manager](https://developer.wordpress.com/apps/).
+After you create an account you can create an application on the [WordPress.com applications manager](https://developer.wordpress.com/apps/).
 
-When creating your application, you should select "Native client" for the application type. The applications manager currently requires a "redirect URL", but this isn't used for mobile apps. Just use "https://localhost".
+When creating your application, select "Native client" for the application type. The applications manager currently requires a "redirect URL", but this isn't used for mobile apps. Just use "https://localhost".
 
-After you created an application you will have an associated a client ID and a client secret key. These details will be used to authenticate your application and verify that the API calls being made are valid. 
+Your new application will have an associated client ID and a client secret key. These are used to authenticate the API calls made by your application. 
 
-In order to use these details, you'll need to create a credential file in your build machine. Start by copying the sample credentials file in your local repo by doing this:
+Next, create a credential file on your build machine. Start by copying the sample credentials file in your local repo by doing this:
```

Comment:
I think rather than "build machine" we could say "development machine" or "local machine"? We don't really have build-specific computers, and I'd imagine neither do our OSS friends? :)

Reply:
Let's remove the specifier completely. It's not really necessary to mention, and without it the instructions are relevant to any machine.